zstacklib/zstacklib/utils/file_metadata_handler.py

```
@@ -275,6 +275,67 @@ def _do_cleanup(self, metadataPath):
+            if not os.path.isfile(fpath):
+                continue
+            try:
+                os.remove(fpath)
+                removed_any = True
+                # Count only the canonical .vmmeta entries; companion files
+                # (.tmp/.summary/.summary.tmp) belong to the same VM and
+                # must not double-count.
+                if fname.endswith(_METADATA_SUFFIX) and \
+                        _UUID_HEX_RE.match(fname[:-len(_METADATA_SUFFIX)]):
+                    cleaned_main += 1
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)